src-java/server42/server42-control/src/main/java/org/openkilda/server42/control/kafka/Gate.java

```diff
@@ -38,13 +41,22 @@
 import org.springframework.kafka.annotation.KafkaHandler;
 import org.springframework.kafka.annotation.KafkaListener;
 import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.kafka.support.KafkaHeaders;
+import org.springframework.messaging.handler.annotation.Header;
+import org.springframework.messaging.handler.annotation.Payload;
 import org.springframework.stereotype.Service;
 
+import java.util.AbstractMap.SimpleEntry;
 import java.util.HashSet;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 @Service
 @Slf4j
-@KafkaListener(id = "server42-control", topics = "${openkilda.server42.control.kafka.topic.from_storm}")
+@KafkaListener(id = "server42-control",
+        topics = "${openkilda.server42.control.kafka.topic.from_storm}",
+        idIsGroup = false
+)
 public class Gate {
```

Comment:
Please, reconsider the name of the class. Or add javadoc. "Gate" is too general to understand the purpose/role.